components/pages/workPageLayout/helper.ts

```diff
@@ -18,8 +21,23 @@ export const getWorkMaterialTypes = (
 
 export const getManifestationMaterialType = (
   manifestation: ManifestationWorkPageFragment
-): WorkMaterialTypesFragment["materialTypes"][0]["materialTypeGeneral"]["display"] => {
-  return manifestation.materialTypes[0].materialTypeGeneral.display
+): WorkMaterialTypesFragment["materialTypes"][0]["materialTypeGeneral"] => {
+  return manifestation.materialTypes[0].materialTypeGeneral
+}
+
+export const getManifestationMaterialTypeSpecific = (
+  manifestation: ManifestationWorkPageFragment
+): "e-bog" | "lydbog" | "podcast" | undefined => {
+  if (isManifestationEbook(manifestation)) {
+    return "e-bog"
+  }
+  if (isManifestationAudioBook(manifestation)) {
+    return "lydbog"
+  }
+  if (isManifestationPodcast(manifestation)) {
+    return "podcast"
+  }
+  return undefined
 }
```

Comment:
I prefer to return `null` in a case like that. By returning `null` you explicitly tell that you have made the choice of doing so. 
So it is like saying:
We don't know how to resolve the type so we return `null` because it is "our" value and not because something slips through the system from an API call. In that way it is possible to distinguish between some thing that has been set deliberately or an unknown value that has not been "catched".
Does it make sense?

Reply:
Yesss I changed it to null :))